rpcs3/Emu/NP/rpcn_client.h

```diff
@@ -437,9 +437,18 @@ namespace rpcn
 		void remove_message_cb(message_cb_func cb_func, void* cb_param);
 		void mark_message_used(u64 id);
 
-		bool is_connected() const;
```

Comment:
I think you can keep all the changes in this file

Reply:
When doing a revert for the purpose of reverting a regression its better to make it a clean revert and not touch anything in the original code.